src/js/tracks/text-track.js

```diff
@@ -227,7 +226,7 @@ TextTrack.prototype.removeCue = function(removeCue) {
 /*
  * Downloading stuff happens below this point
  */
-let parseCues = function(srcContent, track) {
+var parseCues = function(srcContent, track) {
   if (typeof window['WebVTT'] !== 'function') {
```

Comment:
why was this changed? We should move `parseCues` up in the file, if the TDZ is the issue.